app/uk/gov/hmrc/exports/mongock/MongockConfig.scala

```diff
@@ -27,7 +27,7 @@ case class MongockConfig(mongoURI: String) {
 
   val client = new MongoClient(uri)
 
-  val runner = new MongockBuilder(client, uri.getDatabase, "mongock.changesets")
+  val runner = new MongockBuilder(client, uri.getDatabase, "uk.gov.hmrc.exports.mongock.changesets")
```

Comment:
Was the full package name necessary or is this just to be more "accurate".

Reply:
It's necessary, because it's backend service, so backend service has this as the package.

In the frontend application it wasn't necessary, because we don't have packages but directly we're putting stuff inside the `app` folder.

Previous implementation didn't work. I didn't check it (my fault) just check if the application works correctly